### DISCUSSION TITLE
Unable search content from pages

### DISCUSSION THREAD

**Discussion by mu-yadav** [requarks/wiki#6117] [Help / Questions] [answered]
I have added multiple pages on wiki.js . But when I go to any word from those pages , unable to search . It only searches by page name.

**Comment by aurorasmiles** [comment, marked as answer]
The default search (DB-Basic) can only search title and descriptions. To search the content of pages you need to set a different search engine. If you use PostgreSQL you can select DB - PostgreSQL, otherwise you need to use an external service.
See also https://docs.requarks.io/search
